src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,10 @@
 package mate.academy.lib;
 
-public @interface Component {
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 
-}
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Component {}
```

Comment:
this is not used, we should validate if our impl class has it, and if not throw an exception

Reply:
fix